db/migrate/20170519153350_add_showback_models.rb

```diff
@@ -0,0 +1,73 @@
+class AddShowbackModels < ActiveRecord::Migration[5.0]
+  def up
+    create_table :showback_usage_types do |t|
+      t.string     :category
+      t.string     :description
+      t.string     :measure
+      t.text       :dimensions
+      t.timestamps
+    end
+
+    create_table :showback_events do |t|
+      t.jsonb      :data
+      t.timestamp  :start_time
+      t.timestamp  :end_time
+      t.belongs_to :resource, :type => :bigint, :polymorphic => true, :index => true
+      t.jsonb      :context
+      t.timestamps
+    end
+
+    create_table :showback_price_plans do |t|
+      t.string     :name
+      t.belongs_to :resource, :type => :bigint, :polymorphic => true
+      t.string     :description
+      t.timestamps
+    end
+
+    create_table :showback_rates do |t|
+      t.bigint     :fixed_rate_cents # Columns needed by gem money
```

Comment:
In @sergio-ocon 's defense, these particular column types are defined by the money gem, which is what he will be using, so they can't be changed / debated.